EIPS/eip-2981.md

```diff
@@ -16,14 +16,14 @@ A standardized way to retrieve royalty payment information for ERC-721 tokens to
 
 ## Abstract
 
-This standard complements the [ERC-721 specification](./eip-721.md) by enabling an NFT to signal the royalty amount to be paid to the creator or rights holder every time an NFT is sold or re-sold. This is intended for NFT marketplaces that want to support the ongoing funding of artists and other NFT creators. The royalty payment must be voluntary because transfers do not always imply that a sale occurred and enforcement of royalties during a sale can always be defeated. Marketplaces and individuals implement this standard by retrieving the royalty payment information with `royaltyInfo()`, which specifies how much to pay to which address for a given sale price. The exact mechanism for paying and notifying the recipient will be defined in future EIPs. This ERC should be considered a minimal, gas-efficient building block for further innovation in NFT royalty payments.
+This standard complements the [ERC-721 specification](./eip-721.md) and [ERC-1155 specification](./eip-1155.md) by enabling an NFT to signal the royalty amount to be paid to the creator or rights holder every time an NFT is sold or re-sold. This is intended for NFT marketplaces that want to support the ongoing funding of artists and other NFT creators. The royalty payment must be voluntary because transfers do not always imply that a sale occurred and enforcement of royalties during a sale can always be defeated. Marketplaces and individuals implement this standard by retrieving the royalty payment information with `royaltyInfo()`, which specifies how much to pay to which address for a given sale price. The exact mechanism for paying and notifying the recipient will be defined in future EIPs. This ERC should be considered a minimal, gas-efficient building block for further innovation in NFT royalty payments.
```

Comment:
> This standard complements the ERC-721 specification and ERC-1155 specification by enabling...

**Reduce verbosity:**
This standard complements the ERC-721 and ERC-1155 specifications by enabling...